projects/nginx/project.yaml

```diff
@@ -1,2 +1,2 @@
 homepage: "http://nginx.org"
-primary_contact: "xim.andrew@gmail.com"
+primary_contact: "zelenkov@nginx.com"
```

Comment:
Probably add your gmail account as well in auto_ccs, we have a bug where we can't get the alternative email address.